### PR TITLE
Add new Plugs to README

### LINKED ISSUE #74: Add New Plugs to README

### PR DESCRIPTION
**Pull request by doomspork**
Closes #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A project that will render your data models into [JSONAPI Documents](http://json
 [![Build Status](https://travis-ci.org/jeregrine/jsonapi.svg)](https://travis-ci.org/jeregrine/jsonapi)
 
 ## JSONAPI Support
-- [X] Basic [JSONAPI Document](http://jsonapi.org/format/#document-top-level) encoding 
+- [X] Basic [JSONAPI Document](http://jsonapi.org/format/#document-top-level) encoding [issue
```
Comment (jeregrine) [inline README.md]: Sorry I ninja pushed this to master, just rebase and you should be fine.

File: README.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ defmodule MyApp.PostView do
...
 ```
-is an example of a basic view. You can now call `render(conn, MyApp.PostView, "show.json", %{data: my_data})` or `'list.json` normally.
+is an example of a basic view. You can now call `render(conn, "show.json", MyApp.PostView, %{data: my_data})` or `'list.json` normally.
 
 If you'd like to use this without phoenix simply use the `JSONAPI.View` and call `JSONAPI.Serializer.serialize(MyApp.PostView, data, conn)`.
 
 ## Parsing and validating a JSONAPI Request
 
-```elixir
+```elixirg
```
Comment (jeregrine) [inline README.md]: Remove `g`

File: README.md [hunk 3]
```diff
@@ -60,17 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
-## Configuration
```
Comment (jeregrine) [inline README.md]: We should keep this configuration section, possibly at the end.

Comment (doomspork) [reply README.md]: Weird.  I did copy it to the end.  I'm not sure why it's not there.  I'll re-push with your feedback.

**Comment by doomspork** [comment]
Something is really wonky with rebasing against master.  I keep getting strange changes.  Let me reset all of my references and try again.

**Comment by doomspork** [comment]
There we go @jeregrine  👍

**Comment by jeregrine** [comment]
Thank you!! ❤️

**Pull request merged**